### PR TITLE
Change connection status logger to only log on errors

### PR DESCRIPTION
**Pull request by romfrolov**
This PR changes connection status logger to only log on errors.

# Def. of Done
- [ ] Changes described in changelog.md 📝 
- [ ] Automatic tests written 🧪
- [ ] Manually tested 🧪
- [ ] Good code style
- [ ] Necessary migrate instructions added to migrate.md 🚀
- [ ] No known issues

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -7,13 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
-- RPC Server load test and renaming
+### Added
+- RPC Server load test
+
+### Improved
+- Naming
```
Comment (schrodit) [inline changelog.md:14]: can you be bit more specific.
But I think this also is for a different PR?

**Review by schrodit — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 1 above)
Comment (romfrolov) [reply changelog.md:14]: Yeah. It was: "- RPC Server load test and renaming". I'm not sure what was renamed.

**Review by jonas-zipprick — APPROVED** [review]

File: changelog.md (hunk 1 above)
Comment (schrodit) [reply changelog.md:14]: hmm ok. Then maybe lets completely remove it.This change log is as meaningful as not having it.

**Review by schrodit — APPROVED** [review]
I leave the changelog up to you.
Otherwise lgtm

**Pull request merged**